### ISSUE TITLE
FR: normalize output between owa app vs email app

### ISSUE THREAD

**Issue by garanews** [frikky/Shuffle-apps#38]
In order to keep same behavior of the other nodes, it would be useful to have same output of emails app.

**Comment by frikky** [comment]
I agree. This should also happen for the email trigger when it works in the future :+1:

**Comment by frikky** [comment]
Hey @garanews, do you think the new output from the OWA app is good? 

I believe we should use it as a reference then
